go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -332,7 +326,7 @@ func (oc *Controller) updateEgressFirewallWithRetry(egressfirewall *egressfirewa
 }
 
 func (oc *Controller) addEgressFirewallRules(hashedAddressSetNameIPv4, hashedAddressSetNameIPv6, namespace string, efStartPriority int, txn *util.NBTxn) error {
-	ef := oc.namespaces[namespace].egressFirewall
+	ef := oc.egressFirewalls[namespace]
```

Comment:
should we just pass `ef` as a function argument? As in:

```
func (oc *Controller) addEgressFirewallRules(ef ...)
```

Not this PR, ` addEgressFirewallRules` should be a function receiver for `egressFirewall` and not `oc`. Looks like this is done this way so that we can call oc.WatchFactory.Nodes() to get all Node Logical Swtiches for local gateway mode.

Reply:
this will be changed when I make the level triggered controller